### DISCUSSION TITLE
What needs to be done to get macOS support working?

### DISCUSSION THREAD

**Discussion by tanuki-billie** [nzp-team/nzportable#534] [General]
I see that the project supports macOS, but there aren't any premade binaries available, and the nightly builds don't include macOS. I'd like to help - what can be done to get macOS support in the works? There's a billion other things that would have to be considered, but I'd like to get something working quickly.

**Comment by MotoLegacy** [comment]
Thanks for your expressed interest. The main obstacle with macOS support is primarily deployment — working with macOS is a little annoying for me because if I want auto-deployment through CI/CD my only option is to use currently "in-support" revisions, which only includes ARM architectures nowadays. I have a Mojave machine lying about I can use to build slightly older binaries, though that does not solve the deployment problem.
